translations/strings.yaml

```
@@ -55,6 +55,7 @@ serversView:
 
 feedbackForm:
   chooseCategory: Choose a category
+  excellentLabel: Excellent
```

Can we add a comment to provide more context? e.g. 

```suggestion
  excellentLabel: 
    value: Excellent
    comment: 
      - One of the answers on the feedback question about the user
      - experience with the VPN client.
```